src/Support/Contracts/ArgBuilderDirective.php

```diff
@@ -13,10 +11,10 @@ interface ArgBuilderDirective extends Directive
      *
      * TODO try adding a generic type parameter for the type of model when PHPStan handles it better
      *
-     * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder<\Illuminate\Database\Eloquent\Model>|\Illuminate\Database\Eloquent\Relations\Relation<\Illuminate\Database\Eloquent\Model>  $builder  the builder used to resolve the field
-     * @param  mixed  $value  the client given value of the argument
+     * @param Builder $builder the builder used to resolve the field
+     * @param mixed $value the client given value of the argument
```

Comment:
fix formatting in all `@param` please

Reply:
I am using PHPStorm, what is the code style setting you use?

Reply:
Ah or maybe `make fix` does that? Will try later